src/Extensions/PageHealthExtension.php

```diff
@@ -52,8 +53,12 @@ public function updateCMSFields(FieldList $fields)
      */
     public function getRenderedHtml()
     {
+        // use the public URL unless an override is provided, e.g. in a Docker container setup
         if (!$this->renderedHtml) {
-            $this->renderedHtml = file_get_contents($this->getOwner()->AbsoluteLink());
+            $providedInternalServer = Config::inst()->get('Vulcan\Seo\Extensions\PageHealthExtension', 'internal_server_url');
+            $pageURL = !empty($providedInternalServer) ?
+                $providedInternalServer . $this->owner->Link() : $this->getOwner()->AbsoluteLink();
```

Comment:
`Controller::join_links($providedInternalServer, $this->owner->Link())`  will go well here to prevent any issues with double slashes